scripts/prod/zero-downtime-deploy.sh

```
@@ -0,0 +1,238 @@
+        log "Port $port is in use, cleaning up..."
+        kill_process_on_port "$port"
+    fi
+
+    sudo nohup java \
+        -Dspring.profiles.active=$PROFILE,monitor \
+        -Duser.timezone=$TIMEZONE \
+        -Dserver.port=$port \
+        -Dmanagement.server.port=$monitor_port \
+        -Ddd.service=debate-timer \
+        -Ddd.env=$PROFILE \
+        -jar "$jar_file" > "$APP_DIR/app-$port.log" 2>&1 &
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)